loadtest/loadtest.go

```diff
@@ -72,7 +73,14 @@ func (lt *LoadTester) addUser() error {
 	}
 	lt.status.NumUsers++
 	lt.status.NumUsersAdded++
-	controller, err := lt.newController(activeUsers+1, lt.statusChan)
+	// Randomly login with the same user again, to simulate multiple sessions
+	// from the same user.
+	nextUser := rand.Int()%2 == 0
```

Comment:
After this change we are now considering number of sessions instead of number of users right?

Reply:
Basically, yeah.

Reply:
I see, would it be necessary to update method names accordingly?